resolwe/flow/models/data.py

```diff
@@ -486,3 +508,13 @@ class DataDependency(models.Model):
     parent = models.ForeignKey(Data, null=True, on_delete=models.SET_NULL, related_name='children_dependency')
     #: kind of dependency
     kind = models.CharField(max_length=16, choices=KIND_CHOICES)
+
+
+class DataLocation(models.Model):
+    """Location data of the data object."""
+
+    #: data object
+    data = models.OneToOneField(Data, on_delete=models.CASCADE, primary_key=True, related_name='location')
+
+    #: location identifier bound to data object's id
+    location_id = models.CharField(max_length=30, db_index=True)
```

Comment:
`location_id` -> `location`

Reply:
Thinking about this. Even `location` doesn't sound right. Location can mean anything really. It could mean absolute "physical" path, "https" path. What about `directory_name`?

Reply:
That way [this](https://github.com/genialis/resolwe/pull/571/files#diff-1adac73ebf52d60a1bde30c2fceb4610R136) can be more verbose.

Reply:
This is exactly the point here, at some point we will add S3-like storage or something similar and this model has also to cover that case. It can be `location`, `path` or something similar. `location_id` is a bad choice because it is too similar to `location.id`.